taichi/ir/control_flow_graph.cpp

```diff
@@ -256,11 +315,27 @@ void CFGNode::reaching_definition_analysis(bool after_lower_access) {
 
 bool CFGNode::store_to_load_forwarding(bool after_lower_access,
                                        bool autodiff_enabled) {
+  // Contains two separate parts:
+  // 1. Store-to-load Forwarding: for each load stmt, find the closest previous
+  // store stmt
+  //        that stores to the same address and as the load stmt, then replace
```

Comment:
Could this possibly be a typo? Btw the indentation here seems somewhat strange~
```suggestion
  //        that stores to the same address as the load stmt, then replace
```